chirpstack/chirpstack-concentratord-target-rak7268v2/files/chirpstack-concentratord-slot1.config

```diff
@@ -3,12 +3,15 @@ config global
     option chipset          'sx1302'
 
 config sx1302
-    option model            'rak_2287'
+    option model            'rak_5146'
```

Comment:
Are you sure this is correct? The RAK7268v2 is documented to have a SX1302, not SX1303. I can confirm that my gateway has a RAK2287 inside.